src/signals/my-incidents/pages/Overview.test.tsx

```diff
@@ -13,8 +14,44 @@ jest.mock('../components', () => ({
   IncidentsList: () => <div>[IncidentsList]</div>,
 }))
 
+jest.mock('react-router-dom', () => ({
+  __esModule: true,
+  ...jest.requireActual('react-router-dom'),
+  useLocation: () => ({
+    pathname: 'http://www.mijnmeldingen.nl/mijn-meldingen/123kjsbef',
+  }),
+}))
+
+jest.mock('hooks/useFetch')
+
+export const del = jest.fn()
```

Comment:
Ha Vincent,
Voor mijn begrip, waarom spel je hier useFetch zo uit met z'n get, patch, post en put? Ik zie je het niet allemaal gebruiken. Ergens anders in de code staat het soms zoals jij het doet, maar ook soms ook weer niet. Het komt meer dan uitgebreid nodig over, dus euh waarom zo?

Reply:
Because the fetchResponse need to return them. I used the same setup (so writing them all out) as in other places so we can them in a single file in the future. Then all is the same already.